tests/cephadm/test_cephadm_pull.py

```diff
@@ -0,0 +1,70 @@
+import ast
+
+import requests
+import yaml
+
+from ceph.ceph_admin import CephAdmin
+from utility.log import Log
+
+log = Log(__name__)
+
+
+def getting_ceph_version(link):
+    """
+    Method return ceph version from magna002 latest ceph container yaml
+    Args:
+        link: magna002 latest ceph container yaml link
+    Returns:
+        ceph version
+    """
+
+    response = requests.get(link)
+    yaml_response = yaml.safe_load(response.text)
+    ceph_version_5 = []
+    for ceph_version in yaml_response:
+        if ceph_version >= 5:
+            ceph_version_5.append(yaml_response[ceph_version]["ceph-version"])
+    return ceph_version_5[-1]
```

Comment:
Why returning last element ?

Reply:
Thanks @vamahaja
Reason :
In URL always latest build version append at last, Same discuss with Team members before adding the logic.

Reply:
This will fail in case `yaml_response` won't contain any 5.x version. Instead I suggest to use regex in the for loop and return in the same loop once get the required version.

Reply:
As per the discussion, Updated the changes.